### ISSUE TITLE
Failed to remove profile

### ISSUE THREAD

**Issue by pyoor** [MozillaSecurity/ffpuppet#203]
Seen on a site-scout Windows instance.

```
[01:39:01] Failed to remove profile 'C:\Users\TASK_1~1\AppData\Local\Temp\site-scout\ffprofile_p59qhp8c'
[01:39:01] Done.
Traceback (most recent call last):
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\shutil.py", line 617, in _rmtree_unsafe
    os.unlink(fullname)
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\TASK_1~1\\AppData\\Local\\Temp\\site-scout\\ffprofile_p59qhp8c\\storage\\default\\https+++httpsnationalhighwaysafetyadministrationcom.webpush.freshchat.com^partitionKey=%28https%2Cnationalhighwaysafetyadministration.com%29\\cache\\morgue\\26\\{0fd2c4d1-ca4a-41b7-841d-28c9af21eb1a}.final'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\site-packages\site_scout\main.py", line 137, in main
    scout.run(
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\site-packages\site_scout\site_scout.py", line 522, in run
    self._process_active(time_limit)
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\site-packages\site_scout\site_scout.py", line 381, in _process_active
    visit.puppet.close()
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\site-packages\ffpuppet\core.py", line 647, in close
    self.profile.remove(ignore_errors=force_close)
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\site-packages\ffpuppet\profile.py", line 233, in remove
    rmtree(self.path, onerror=onerror)
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\shutil.py", line 749, in rmtree
    return _rmtree_unsafe(path, onerror)
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\shutil.py", line 614, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\shutil.py", line 614, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\shutil.py", line 614, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  [Previous line repeated 3 more times]
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\shutil.py", line 619, in _rmtree_unsafe
    onerror(os.unlink, fullname, sys.exc_info())
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\site-packages\ffpuppet\helpers.py", line 253, in onerror
    chmod(path, S_IWUSR)
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\TASK_1~1\\AppData\\Local\\Temp\\site-scout\\ffprofile_p59qhp8c\\storage\\default\\https+++httpsnationalhighwaysafetyadministrationcom.webpush.freshchat.com^partitionKey=%28https%2Cnationalhighwaysafetyadministration.com%29\\cache\\morgue\\26\\{0fd2c4d1-ca4a-41b7-841d-28c9af21eb1a}.final'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "C:\Users\task_171625028602581\msys64\opt\python\lib\runpy.py", line 196, in _run_module_as_main
```

**Comment by pyoor** [comment]
https://community-tc.services.mozilla.com/tasks/VsRG6aMkRcS80FyB_iXxKQ/runs/0/logs/project/fuzzing/private/logs/live.log#L456

**Comment by tysmith** [comment]
Fixed by #205
